config/rbac/role.yaml

```diff
@@ -156,3 +156,24 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - policy
+  resources:
+  - poddisruptionbudgets
+  verbs:
+  - list
+  - watch
+- apiGroups:
+  - policy
+  resourceNames:
+  - eventing-nats
+  resources:
+  - poddisruptionbudgets
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
Do we need all these permissions?

Reply:
yes. We do the server-side patch apply. Anyway the role is bind to resource name:
```
resourceNames:
  - eventing-nats
```